testcases/OpTestEM.py

```diff
@@ -434,25 +434,25 @@ def runTest(self):
             self.disable_idle_state(i)
             self.verify_disable_idle_state(i)
 
-        # With all idle disabled, gather current usage (as a baseline)
-        before_usage = {}
+        # With all idle disabled, gather total time spent in this idle state (as a baseline)
+        before_time = {}
         for i in idle_states:
-            before_usage[i] = self.c.run_command("cat /sys/devices/system/cpu/cpu*/cpuidle/state%s/usage" % (i))
-            before_usage[i] = [int(a) for a in before_usage[i]]
+            before_time[i] = self.c.run_command("cat /sys/devices/system/cpu/cpu*/cpuidle/state%s/time" % (i))
+            before_time[i] = [int(a) for a in before_time[i]]
 
-        after_usage = {}
+        after_time = {}
```

Comment:
Can we keep both time and usage, so that we can see both stats. As usage will tell us how many times we entered each stop state.